e2e-tests/cypress/integration/skills_spec.js

```diff
@@ -711,52 +711,108 @@ describe('Skills Tests', () => {
+
+        cy.visit('/administrator/projects/proj1/subjects/subj1/skills/skill1');
+        cy.wait('@loadSkill1');
+
+        cy.contains('SKILL: This is 1').should('be.visible');
+        cy.get('[data-cy=childRowDisplay_skill1]').should('be.visible');
+        // skill should now only be loaded once on page load instead of twice, once by SkillPage and another time by SkillOverview
+        cy.get('@loadSkill1.all').should('have.length', 1);
+        cy.get('[data-cy=editSkillButton_skill1]').click();
+        cy.get('input[data-cy=skillName]').type('{selectall}Edited Skill Name');
+        cy.get('input[data-cy=skillPointIncrement]').click();
+        cy.get('[data-cy=saveSkillButton]').click();
+        cy.wait('@saveSkill1');
+        cy.get('[data-cy=editSkillButton_skill1]').should('have.focus');
+        cy.contains('SKILL: Edited Skill Name').should('be.visible');
+        cy.contains('SKILL: This is 1').should('not.exist');
+
+        cy.get('[data-cy=breadcrumb-skill1]').should('be.visible');
+        cy.get('[data-cy=editSkillButton_skill1]').click();
+        cy.get('[data-cy=idInputEnableControl] a').click();
+        cy.get('input[data-cy=idInputValue]').type('{selectall}entirelyNewId');
+        cy.get('input[data-cy=skillPointIncrement]').click();
+        cy.get('[data-cy=saveSkillButton]').click();
+        cy.wait('@validateDescription');
+        cy.wait('@validateUrl');
+        cy.wait('@afterIdEdit');
+        cy.get('[data-cy=editSkillButton_entirelyNewId]').should('have.focus');
+        cy.contains('ID: entirelyNewId').should('be.visible');
+        cy.get('[data-cy=breadcrumb-skill1]').should('not.exist');
+        cy.get('[data-cy=breadcrumb-entirelyNewId]').should('be.visible');
+        cy.get('[data-cy=editSkillButton_entirelyNewId]').should('be.visible');
+        cy.get('[data-cy=editSkillButton_skill1]').should('not.exist');
+
+        //edit version, point increment, occurrences, time window, description, helpurl and confirm that the updates are reflected in the overview section
+        cy.get('[data-cy=editSkillButton_entirelyNewId]').click();
+        cy.get('[data-cy=skillPointIncrement]').type('{selectall}20');
+        cy.get('[data-cy=saveSkillButton]').click();
+        cy.wait('@validateDescription');
+        cy.wait('@validateUrl');
+        cy.wait('@afterIdEdit');
+        cy.get('[data-cy=skillOverviewTotalpoints]').contains('100 Points').should('be.visible');
+        cy.get('[data-cy=skillOverviewTotalpoints]').contains('20 points').should('be.visible');
+        cy.get('[data-cy=skillOverviewTotalpoints]').contains('5 repetitions to Completion').should('be.visible');
+        cy.get('[data-cy=editSkillButton_entirelyNewId]').click();
+        cy.get('[data-cy=numPerformToCompletion]').type('{selectall}10');
+        cy.get('[data-cy=saveSkillButton]').click();
+        cy.wait('@validateDescription');
+        cy.wait('@validateUrl');
+        cy.wait('@afterIdEdit');
+        cy.wait(200);
```

Comment:
tried without timed waits and it worked, looks like timed waits can now be removed